PKSim.sln

```diff
@@ -106,8 +106,8 @@ Global
 		HideSolutionNode = FALSE
 	EndGlobalSection
 	GlobalSection(ExtensibilityGlobals) = postSolution
-		SolutionGuid = {13620E23-1993-42C6-A4AB-468E29DD24DE}
 		VisualSVNWorkingCopyRoot = 
+		SolutionGuid = {13620E23-1993-42C6-A4AB-468E29DD24DE}
```

Comment:
undo this change here. This is some Local SVN stuff that we do not use

Reply:
The only change here ist that the lines 109 and 110 were interchanged (for whatever reason), nothing new was added.

Reply:
ok. I thought the line with SVN was added. Cool then